pulp_container/tests/functional/api/test_sync.py

```diff
@@ -245,13 +256,12 @@ def sync_repository_with_whitelisted_tags(self, whitelist_tags):
         sync_response = self.repositories_api.sync(self.repository.pulp_href, repository_sync_data)
         monitor_task(sync_response.task)
         repository = self.repositories_api.read(self.repository.pulp_href)
-        self.assertIsNotNone(repository.latest_version_href)
 
-    def assert_synced_tags(self, whitelist_tags):
-        """Check if the created repository contains only the selected whitelisted tags."""
+    def assert_synced_tags(self, include_tags):
```

Comment:
what about adding a test where same tag is present in both places?
I think then this assertion will need to be changed because in this case list of tags wont be equal to `include_tags`

Reply:
The 'complex filtering' test is doing this?
Maybe the name of this parameter is misleading. I can rename it to `expected_tags`.

Reply:
right, i see that now! +1 to rename